initrd/directory.go

```diff
@@ -17,6 +17,8 @@ import (
 	"kraftkit.sh/log"
 )
 
+const RootfsOutputAlreadyExist = "rootfs output already exists"
```

Comment:
Let's declare this as an error, this is better standard practice:
```diff
- const RootfsOutputAlreadyExist = "rootfs output already exists"
+ var ErrRootfsOutputAlreadyExists = fmt.Errorf("rootfs output already exists")
```

Reply:
Done 👍🏽